apps/web/app/future/settings/(settings)/organizations/profile/page.tsx

```diff
@@ -1,11 +1,27 @@
-import { _generateMetadata } from "app/_utils";
+import { _generateMetadata, getFixedT } from "app/_utils";
 
-import Page from "@calcom/features/ee/organizations/pages/settings/profile";
+import { getServerSessionForAppDir } from "@calcom/features/auth/lib/get-server-session-for-app-dir";
+import LegacyPage from "@calcom/features/ee/organizations/pages/settings/profile";
+import SettingsHeader from "@calcom/features/settings/appDir/SettingsHeader";
 
 export const generateMetadata = async () =>
   await _generateMetadata(
     (t) => t("profile"),
     (t) => t("profile_org_description")
   );
 
+const Page = async () => {
+  const session = await getServerSessionForAppDir();
+  const t = await getFixedT(session?.user.locale || "en");
```

Comment:
I wonder if we should move this to a UTIL to get translations for active user in appdir

Reply:
Yeah, good point. Can you create a separate PR for that?